blockmanager.go

```diff
@@ -279,6 +297,34 @@ func newBlockManager(cfg *blockManagerCfg) (*blockManager, error) {
 	}
 	bm.filterHeaderTipHash = fh.BlockHash()
 
+	// If sideload is enabled initialize reader and assign to blockmanager.
+	if bm.cfg.sideLoad.Enabled {
+		log.Infof("Side loading enabled")
+
+		reader, err := chaindataloader.NewBlockHeaderReader(&chaindataloader.ReaderConfig{
```

Comment:
It should error out and fail if this was specified, but we weren't able to actually read it.

Reply:
Fail? Like the `newBlockManger` function should return an error? The design is to skip sideloading and carry on with the normal p2p sync if there's an error, sideloading is not a critical part of the neutrino. The error is logged though.

Reply:
See prior comments: I think this should be done at start up, before the block manager is even active itself. Then that way we don't need to worry about what to do if this fails while we're still trying to load other blocks.

Reply:
I think we need the blockmanager to be active, we still need to fetch cfheaders. I do not think fetching cfheaders should halt because we are fetching blockheaders from a non-p2p source. They are both two distinct processes. What do you think?

Reply:
Well we can _also_ have the cfheaders be ingested in a flat file. Even before that, the with the way things are set up, we first sync the block headers to the next milestone _then_ sync the filter headers. As a result, if you start with all the block headers already loaded, _then_ start up, the system will work as expected with the filters headers being synced in the background. 

In other words, fetching filter headers won't halt.